Example/android/gradle.properties

```diff
@@ -19,4 +19,5 @@
 android.useAndroidX=true
 android.enableJetifier=true
 # Version of flipper SDK to use with React Native
-FLIPPER_VERSION=0.37.0
\ No newline at end of file
+FLIPPER_VERSION=0.37.0
+ANDROID_NDK_VERSION=20.1.5948944
```

Comment:
We have the same configuration variable in two places, here and [here](https://github.com/software-mansion/react-native-reanimated/blob/f7b140a9c379262d58f7051aef3bde658a987879/android/gradle.properties#L22). Is it possible to set NDK version in one place?

Maybe we should add to docs info about the required NDK version, what do you think?

Reply:
It's not used right now. I will remove it.